tests/acceptance/fixtures.py

```diff
@@ -253,11 +253,11 @@ def cleanup_test_build():
     local_conf = os.path.join(build_dir, "conf", "local.conf")
     local_conf_orig = local_conf + ".orig"
     with open(local_conf, "a") as fd:
-        fd.write('SSTATE_MIRRORS = " file://.* file://%s/sstate-cache/PATH"\n' % os.environ['BUILDDIR'])
+        fd.write('SSTATE_MIRRORS = " file://.* file://%s/PATH"\n' % bitbake_variables['SSTATE_DIR'])
+        fd.write('SSTATE_DIR = "${TOPDIR}/sstate-cache"\n')
+        fd.write('DL_DIR = "%s"\n' % bitbake_variables['DL_DIR'])
```

Comment:
I'm probably missing something obvious, but I don't understand how it is supposed to work. Isn't the `SSTATE_DIR` assignment a no-op, IOW it is the same location as the default? `${TOPDIR}` refers to the build directory, which will be the test-specific directory in this case right?

Otherwise I'm all for the cause! :-)

Reply:
Indeed.  I am fairly certain I intended to use 'bitbake_variables['SSTATE_DIR']' here so it will properly pull that, if it is modified (which I do locally to share one sstate directory across all my branches and builds).  I'll take a look and see if I can determine whether this is just an oversight on my part.

Reply:
Actually, I remember now. The tests use the existing SSTATE_DIR has an SSTATE_MIRROR to reuse them without polluting them with testing-based artifacts.  I think I put SSTATE_DIR here out of a sense of completion but you're right, it should be left at the default in case it changes.